pylops/signalprocessing/Interp.py

```diff
@@ -16,23 +17,23 @@ def _checkunique(iava):
         raise ValueError("Repeated values in iava array")
 
 
-def _nearestinterp(dims, iava, dir=0, dtype="float64"):
+def _nearestinterp(dims, iava, axis=0, dtype="float64"):
     """Nearest neighbour interpolation."""
     iava = np.round(iava).astype(int)
     _checkunique(iava)
-    return Restriction(dims, iava, dir=dir, dtype=dtype), iava
+    return Restriction(dims, iava, axis=axis, dtype=dtype), iava
 
 
-def _linearinterp(dims, iava, dir=0, dtype="float64"):
```

Comment:
Not a big thing but we can put also here axis=-1 (and in the _sincinterp :)

Reply:
Yes! Done.